osc-check_source.py

```diff
@@ -257,6 +257,7 @@ def _checker_one_request(self, rq, opts):
             self._devel_projects['filesystems:ceph/'] = 'ceph-deploy'
             self._devel_projects['Virtualization:Appliances:Builder/'] = 'python3-kiwi'
             self._devel_projects['isv:ownCloud:desktop/'] = 'owncloud-client'
+            self._devel_projects['system:snappy/'] = 'snappy'
```

Comment:
We have like 10 elements now, maybe time to put it in a JSON document? Or in a dict in the beginning of the file?